cpp/test/compat/grpc/pingpong_server.cpp

```diff
@@ -106,6 +105,9 @@ int main()
 
     bool countdownSet = Countdown.wait(std::chrono::seconds(30));
 
+    server->Shutdown();
```

Comment:
Consider using the overload that takes a timeout after which it will forcefully fail the in-flight RPC operations.